tests/dashbio_demos/app_circos.py

```diff
@@ -145,414 +907,290 @@ def circos_explain():
 
 # Upload text blurb
 upload_instructions = (
-    "1. Select your dataset or (press download for sample data). \n"
-    + "2. Drag and drop .CSV for each dataset dropdown (layout -> layout.csv, etc) \n"
-    + "3. Press Render! \n"
-    + "4. Go to 'View Dataset' tab to view data in table."
+    '1. Select your dataset or (press download for sample data). \n'
+    + '2. Drag and drop .CSV for each dataset dropdown (layout -> layout.csv, etc) \n'
```

Comment:
```suggestion
    + '2. Drag and drop .csv for each dataset dropdown (layout -> layout.csv, etc) \n'
```
I write either "CSV files" or ".csv files" (we may want to be extra robust above by using `filename.lower().endswith('csv')`...).